lib/membrane_portaudio_plugin/source.ex

```diff
@@ -20,7 +20,7 @@ defmodule Membrane.PortAudio.Source do
     flow_control: :push,
     accepted_format: %RawAudio{sample_format: format} when format in @sample_formats
 
-  def_options endpoint_id: [
+  def_options device_id: [
```

Comment:
I'd leave the old option, but document it as deprecated and warn in `handle_init`